src/components/CalendarPicker/index.js

```diff
@@ -5,31 +5,28 @@ import moment from 'moment';
 import Text from '../Text';
 import ArrowIcon from './ArrowIcon';
 import styles from '../../styles/styles';
-import {propTypes, defaultProps} from './calendarPickerPropTypes';
+// eslint-disable-next-line rulesdir/prefer-import-module-contents
+import {calendarPickerPropType, defaultCalendarPickerPropType} from './calendarPickerPropTypes';
```

Comment:
As note before;
NAB: We usually export those as propTypes, defaultProps and import them as calendarPickerPropTypes, calendarPickerDefaultProps.

Is the eslint rule skip required here?

Reply:
+1